packages/blockchain-wallet-v4-frontend/src/modals/Brokerage/Banks/Withdraw/WithdrawalMethods/BankTransfer/index.tsx

```diff
@@ -54,10 +53,25 @@ const BankTransfer = ({ icon, onClick, value }: Props) => (
         <DisplayIconBank>{icon}</DisplayIconBank>
         <Content>
           <DisplayTitleBank>
-            <FormattedMessage id='copy.bank_account' defaultMessage='Bank Account' />
+            <FormattedMessage id='copy.bank_account' defaultMessage='Easy Bank Transfer' />
           </DisplayTitleBank>
           <DisplaySubTitle>
-            <FormattedMessage id='copy.instantly_available' defaultMessage='Instantly Available' />
+            {value.currency === 'USD' ? (
+              <FormattedMessage
+                id='copy.arrive_in_3_business_days'
+                defaultMessage='Should arrive in 3 business days'
+              />
+            ) : value.currency === 'GBP' || value.currency === 'EUR' ? (
+              <FormattedMessage
+                id='copy.arrive_in_3_business_days'
+                defaultMessage='Should arrive instantly'
+              />
+            ) : (
+              <FormattedMessage
+                id='copy.instantly_available'
+                defaultMessage='Instantly Available'
+              />
+            )}
```

Comment:
Same case as one of the previous comments. Nested ternaries should be renamed to nasty ternaries.